src/System.Text.Formatting/System/Text/Parsing/InvariantParser_ulong.cs

```diff
@@ -46,5 +46,41 @@ public static bool TryParse(Utf8String text, out ulong value, out int bytesConsu
 
             return true;
         }
+		public static bool TryParse(byte[] text, out ulong value, out int bytesConsumed)
+        {
```

Comment:
Ah, so the byte array just represents some buffer passed to us then, not necessarily the specific string we want to parse -- do I have that right? Will add that in then.